turbine/src/broadcast_stage/broadcast_utils.rs

```diff
@@ -28,32 +28,47 @@ fn data_shred_bytes_per_batch() -> u64 {
     *get_data_shred_bytes_per_batch_typical()
 }
 
-fn target_batch_bytes_default() -> u64 {
-    // Empirically discovered to be a good balance between avoiding padding and
-    // not delaying broadcast.
-    3 * data_shred_bytes_per_batch()
+static TARGET_BATCH_BYTES_DEFAULT: OnceLock<u64> = OnceLock::new();
+pub fn get_target_batch_bytes_default() -> &'static u64 {
+    TARGET_BATCH_BYTES_DEFAULT.get_or_init(|| {
+        // Empirically discovered to be a good balance between avoiding padding and
+        // not delaying broadcast.
+        3 * data_shred_bytes_per_batch()
+    })
 }
 
-fn target_batch_pad_bytes() -> f64 {
-    // Less than 5% padding is acceptable overhead. Let's not push our luck.
-    data_shred_bytes_per_batch() as f64 * 0.05
+static TARGET_BATCH_PAD_BYTES: OnceLock<u64> = OnceLock::new();
+pub fn get_target_batch_pad_bytes() -> &'static u64 {
+    TARGET_BATCH_PAD_BYTES.get_or_init(|| {
+        // Less than 5% padding is acceptable overhead. Let's not push our luck.
+        (data_shred_bytes_per_batch() as f32 * 0.05) as u64
```

Comment:
nit: would maybe just recommend division by 20 here to avoid additional casting? definitely won't die on this hill.
```suggestion
        data_shred_bytes_per_batch() / 20
```

Reply:
done